assignments/objects.js

```diff
@@ -49,16 +104,36 @@ const example = {
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-const parent = {}
-
+const parent ={
+    name:'Susan',
+    age: 70,
+    child : {
+       name:'george',
+       age:50,
+   
+
+   grandchild: {
```

Comment:
Great job nesting but I would use tab to indicate depth rather than these big white spaces.